### PR TITLE
Push it

### PR DESCRIPTION
**Pull request by lhh**

### CODE REVIEW

File: errata_tool/erratum.py [hunk 1]
```diff
@@ -888,6 +888,28 @@ def commit(self):
...
+                                 '\'stage\', \'live\', or a list')
+
+            r = self._post(url + '?defaults=' + str(target))
+            self._processResponse(r)
+            return True
+        if not isinstance(target, list):
+            raise ValueError('Wrong value for target: expected ' +
+                             '\'stage\', \'live\', or a list')
+        r = self._post(url, data=target)
+        self._processResponse(r)
```
Comment (ktdreyer) [inline errata_tool/erratum.py]: Is there any point to returning the response here? (does `r` have anything useful?)

File: errata_tool/cli/advisory.py [hunk 2]
```diff
@@ -46,18 +46,32 @@ def add_parser(subparsers):
                                help='eg. "ohno@redhat.com"')
     create_parser.set_defaults(func=create)
 
+    # "push"
+    get_parser = sub.add_parser('push')
+    get_parser.add_argument('errata_id', help='advisory id, "12345"')
+    get_parser.add_argument('--target', choices=('stage', 'live'),
+                            default='stage',
+                            help='stage (default) or live')
+    get_parser.set_defaults(func=push)
```
Comment (ktdreyer) [inline errata_tool/cli/advisory.py]: rename to "push_parser" ?

File: errata_tool/erratum.py (hunk 1 above)
Comment (lhh) [reply errata_tool/erratum.py]: Raises exceptions when things blow up, same as other code paths?

File: errata_tool/cli/advisory.py (hunk 2 above)
Comment (lhh) [reply errata_tool/cli/advisory.py]: Whooops :P

File: errata_tool/erratum.py (hunk 1 above)
Comment (lhh) [reply errata_tool/erratum.py]: I misread what you asked - yes, returning the JSON response makes total sense.

Comment (ktdreyer) [reply errata_tool/erratum.py]: Looks great!

**Pull request merged**